api/server.py

```diff
@@ -42,6 +45,32 @@
 
 ADMIN_ROLE_NAME = "vz-admin"
 
+CORS_URL = "*"
+ALGORITHMS = ["RS256"]
+APP = Flask(__name__)
+
+
+def get_secure_password(num_chars=16):
+    """Generate a secure random password with at least one lowercase character,
+    uppercase character, special character, and digit:
+    https://docs.python.org/3/library/secrets.html#recipes-and-best-practices
+
+    Args:
+        num_chars (int, optional): Defaults to 16.
+    """
+    special_chars = "!@#$%^&*"
+    alphabet = string.ascii_letters + string.digits + special_chars
+    while True:
+        password = "".join(secrets.choice(alphabet) for i in range(num_chars))
+        if (
+            any(c.islower() for c in password)
+            and any(c.isupper() for c in password)
+            and any(c.isdigit() for c in password)
+            and any(c in special_chars for c in password)
+        ):
+            break
+    return password
```

Comment:
It's A+. The whole model of not letting the account creator know the initial password of the account receiver is a step above. It's really fine, thank you!